packages/ui/src/components/DatePicker/SDatePickerTableDate.vue

```diff
@@ -0,0 +1,424 @@
+  const idx = target.getAttribute('index')
+  if (!idx) return
+  const cell = dateTableCells.value[idx]
+  const newDate = new Date(year.value, cell.month, cell.text)
+  if (isSameDay(newDate, props.hoveredDate)) return
+  emit('updateHoveredDate', newDate)
+}
+
+const handleClick = (ev: any) => {
+  let target = ev.target
+  if (target.tagName === 'SPAN') {
+    target = target.parentNode
+  }
+  if (target.tagName !== 'DIV') return
+  const idx = target.getAttribute('index')
+  if (!idx) return
+  const cell = dateTableCells.value[idx]
+  const newDate = new Date(year.value, cell.month, cell.text)
+  if (type === 'range') {
+    if (!props.stateStore.rangeState.selecting) {
+      emit('pick', {
+        startDate: newDate,
+        endDate: null,
+        selecting: true,
+        selectedField: 'startDate',
+      })
+      lastField.value = 'startDate'
+    } else {
+      const dat = props.stateStore.rangeState.startDate || props.stateStore.rangeState.endDate
+      if (dat && newDate >= dat) {
+        emit('pick', {
+          startDate: dat,
+          endDate: newDate,
+          selecting: false,
+          selectedField: 'endDate',
+        })
+        lastField.value = 'endDate'
+      } else {
+        emit('pick', {
+          startDate: newDate,
+          endDate: dat,
+          selecting: false,
+          selectedField: 'startDate',
+        })
+        lastField.value = 'startDate'
+      }
+    }
+  } else if (type === 'day') {
+    emit('pick', newDate)
+  } else if (type === 'pick') {
+    const currentArr = props.stateStore.pickState
+    const dateSelected = currentArr.find((date: Date) => {
+      return isSameDay(date, newDate)
+    })
+    let newArr
+    if (dateSelected) {
+      newArr = currentArr.filter((date: Date) => !isSameDay(date, newDate))
+    } else {
+      newArr = [...currentArr, newDate]
+    }
+    emit('pick', newArr)
+  }
+
+  if (cell.type === 'prev-month') {
+    emit('updateShowedState', -1)
+  }
+  if (cell.type === 'next-month') {
+    emit('updateShowedState', 1)
+  }
+}
+</script>
+<template>
+  <div
+    class="s-date-picker-date-table date-table sora-tpg-p4"
+    @click="handleClick"
+    @mouseover="handleMouseMove"
+    @focus="handleMouseMove"
+    @keydown="handleClick"
+  >
+    <div
+      v-for="(day, key) in weeks"
+      :key="key"
+      class="date-table__title"
+    >
+      {{ day }}
+    </div>
+    <div
+      v-for="(cell, idx) in dateTableCells"
+      :key="idx"
+      class="date-table__cell relative"
+      :class="getCellClasses(cell)"
+      :index="idx"
+    >
+      <span>{{ cell.text }}</span>
+      <span
+        v-if="time"
+        v-show="cell.time"
+        class="absolute date-time"
+      >{{ cell.time }}</span>
+    </div>
+  </div>
+</template>
+
+<style lang="scss">
+@use '@/theme';
+
+.s-date-picker-date-table {
+  font-size: 12px;
+  -webkit-user-select: none;
+  -moz-user-select: none;
+  -ms-user-select: none;
+  user-select: none;
```

Comment:
```suggestion
  user-select: none;
```

I think PostCSS does it for us.

Reply:
But I am not sure. Anyway, we can plug AutoPrefixer with Vite later.